k8s/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:7 as base
```

Comment:
is this image is going to be uploaded to dockerhub ?

Reply:
i.e. you have plan on how to publish this docker image ? versions and such ?

Reply:
I haven't thought about it, I decided to use what you guys use which is output of `SCYLLA-VERSION-GEN`. Check build_image script for details.

Reply:
After we merged that should be thought about...